### DISCUSSION TITLE
Why casaos will auto jion a public zerotier network which was likely created by casaos team?

### DISCUSSION THREAD

**Discussion by LSP0123** [IceWhaleTech/CasaOS#1397] [Q&A]
![image](https://github.com/IceWhaleTech/CasaOS/assets/31061624/9b9eb7ce-bf45-4c03-b412-3f0f1f353a30)

what did you do!!! casaos team?

![image](https://github.com/IceWhaleTech/CasaOS/assets/31061624/9252647c-a6a4-4948-ad22-f531567244a7)

**Comment by clintonmarshall** [comment]
I too just noticed an unauthorised network added to my CasaOS installs. Please explain why this was done without communicating, and why there is no documentation on its purpose.

**Reply by LSP0123** [reply]
I have get answer on their Chinese community。

Their explanation for this is that the network's controllers are built on this machine for yourself and do not pose a security problem for your system. 

You can see  the config on  `/var/lib/zerotier-one/controller.d/network` ,If you don't need it,  you can stop zerotier service and delete  `/var/lib/zerotier-one/controller.d/` 

like this：
```
systemctl stop zerotier-one
cd /var/lib/zerotier-one/
# rm -rf ./controller.d/
systemctl start zerotier-one
```

But the controller will rebuild when you reboot your system。 

Casaos Team said that the IceWhale-RemoteAccess of zerotier will be delete in the next version。you can see [zerotier manual](https://docs.zerotier.com/self-hosting/network-controllers#clean-up-networks ) for more infomation

@JohnGuan Can you indicate where the relevant code is？




For Chinese user：
![image](https://github.com/IceWhaleTech/CasaOS/assets/31061624/d7f6e189-1690-4e61-8a23-dd9ce986d300)
![image](https://github.com/IceWhaleTech/CasaOS/assets/31061624/7e0fb20c-7fa2-44f8-9927-62fa87db379f)
